app/res/values-v14/themes.xml

```diff
@@ -4,36 +4,10 @@
         Base application theme for API 14+. This theme completely replaces
         AppBaseTheme from res/values/styles.xml on API 14+ devices.
     -->
-    <style name="AppBaseTheme" parent="android:Theme.Holo.Light">
+    <style name="AppBaseTheme" parent="CommonTheme">
         <!-- API 14 theme customizations can go here. -->
-        <item name="android:colorBackground">@color/cc_neutral_bg</item>
```

Comment:
confirming these have been removed since they are same as v-11 values ?

Reply:
We can drop v-11 now actually insted of moving these to v-11 since min supported version is 14.

Reply:
or maybe we can use the v-14 as the default one.

Reply:
Not sure if I unserstand the difference correctly though won't v-14 will become the default one by itself once we drop v11 ?

Reply:
We have a different `AppBaseTheme` style in the current default `themes.xml` which would never be used, considering we're only supporting v14+ devices. 
So, we can just replace the default `themes.xml` contents with the contents of themes.xml(v-14) instead of maintaining 2 different themes, right?

Reply:
yup, got it. That does seem like the right behaviour to me. Would definitely love to get rid of the extra theme files.